test/suites/dev/moonbase/test-randomness/test-randomness-babe-lottery3.ts

```diff
@@ -21,25 +21,25 @@ describeSuite({
   foundationMethods: "dev",
   testCases: ({ context, it, log }) => {
     let lotteryAddress: `0x${string}`;
+    let estimatedGasBefore;
 
     beforeEach(async function () {
       lotteryAddress = await setupLotteryWithParticipants(context, "BABE");
 
-      const estimatedGas = await context.viem().estimateContractGas({
+      estimatedGasBefore = await context.viem().estimateContractGas({
```

Comment:
This change seems not related to tracing tests

Reply:
This was giving errors: `Error: toMatchInlineSnapshot cannot be called multiple times at the same location`.

Which makes sense, it is not correct to call it in `beforeEach`

Reply:
Can you update PR title and description then, because this PR is definitely not related to tracing tests only

Reply:
Done